src/transformers/cache_utils.py

```diff
@@ -958,8 +968,13 @@ def update(
         k_out = k_out[:, :, indices]
         v_out = v_out[:, :, indices]
 
-        k_out[:, :, cache_position] = key_states
-        v_out[:, :, cache_position] = value_states
+        try:
```

Comment:
in the final commit, ensure that the two sets of changes are the same (in the current commit, this one is missing some logic)

Reply:
you are right, I forgot it!